assets/javascripts/discourse/templates/docs-topic-link.hbr

```diff
@@ -1 +1 @@
-<a href="/docs?topic={{topic.id}}" class="docs-topic-link">{{{topic.fancyTitle}}}</a>
+<a href="/{{urlPath}}?topic={{topic.id}}" class="docs-topic-link">{{{topic.fancyTitle}}}</a>
```

Comment:
Not a change in this PR, but does this URL starting with a `/` cause issues in a subfolder setup? Not necessary to fix in this PR, but would be good to know.

Reply:
I'll figure out a way to check this locally and will report back.